src/Rheologies/elasto_visco_plastic_rheology.jl

```diff
@@ -12,13 +12,17 @@ using KernelAbstractions: @kernel, @index
 # Where:
 # σᵢⱼ(u) = 2η ϵ̇ᵢⱼ + [(ζ - η) * (ϵ̇₁₁ + ϵ̇₂₂) - P / 2] δᵢⱼ
 #
-struct ElastoViscoPlasticRheology{FT}
+struct ElastoViscoPlasticRheology{RP, FT}
```

Comment:
maybe better to introduce a property called `pressure_formulation`. This is more extensible (eg more than two options), and avoids the complexity of a floating type parameter